### DISCUSSION TITLE
No module named 'diffusers.models.unet_2d_condition'

### DISCUSSION THREAD

**Discussion by NomadBear7** [open-mmlab/mmediting#2147] [How-to]
在运行代码的过程中出现了这个报错是咋回事No module named 'diffusers.models.unet_2d_condition'

**Comment by doubleZ0108** [comment]
同样遇到了这个问题，请问有解决方案了吗？

**Comment by SolveProb** [comment]
修改 mmagic\models\editors\vico\cico_utils.py line 10 为：
from diffusers.modes.unets.unet_2d_condition import UNet2DConditionOutput

ref: https://github.com/huggingface/diffusers/discussions/8866

**Comment by xx-tao** [comment]
pr: https://github.com/open-mmlab/mmagic/pull/2150/changes/c2f0278dff5d3d7c8dfe6f36d7d53b448dd69857
